### PR TITLE
Creation of crowdin file

### PR DESCRIPTION
**Pull request by daniserejo**
#### What problem is this solving?

Creation of crowdin file

### CODE REVIEW

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]
Something is strange with Changelog.

**Review by camilavcoutinho — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Crowdin file.
+
```
Comment (igorbrasileiro) [inline CHANGELOG.md]: Not necessary, this PR is just for tooling.
```suggestion
```

**Review by igorbrasileiro — DISMISSED** [review]
Remove changelog

File: .vscode/settings.json [hunk 2]
```diff
@@ -31,5 +31,8 @@
       "directory": "node",
       "changeProcessCWD": true
     }
-  ]
+  ],
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
```
Comment (igorbrasileiro) [inline .vscode/settings.json]: These changes are intended to? Maybe it's just an editor setting. Remove it

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,15 +8,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ## [0.1.3] - 2020-03-24
+
```
Comment (igorbrasileiro) [inline CHANGELOG.md:11]: Remove these changes.

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**